### DISCUSSION TITLE
추천페이지 예외상황 공유

### DISCUSSION THREAD

**Discussion by Haroldfromk** [SwiftBreakers/TTEOPPOKKI4U#43] [General]
현재 추천페이지에서 깡통시장 관련 글이 새로 추가된것 같은데 지도를 클릭하면 api호출 결과가 없는지 out of range 에러가 발생합니다.
확인 부탁드립니다.

또한 현재 지도를 한번도 클릭하지 않은상태에서 추천페이지를 통해 지도를 클릭하게되면 첫 지도화면이 나옵니다.
이후 다시 클릭하면 위치로 이동이 되는 예외 상황이 있습니다.

![Jun-15-2024 21-33-09](https://github.com/SwiftBreakers/TTEOPPOKKI4U/assets/97341336/ea03b1d7-cf99-4023-ad3b-e4fff31970ac)

관련 페이지 담당하시는분들은 확인 부탁드릴게요

**Comment by jinmoon23** [comment]
헉 얼마전에 확인했을때는 잘 가던 것 같던데.. 확인해보겠습니다!

**Reply by Haroldfromk** [reply]
처음에 지도를 활성화 하지않으면 주소가 이동이 안되는것같아요.

**Comment by jinmoon23** [comment]
https://github.com/SwiftBreakers/TTEOPPOKKI4U/assets/161567425/bf00acb3-e567-40d6-a533-e188dac42bbc


정상작동 확인됩니다!
